builtin/mainmenu/dlg_config_world.lua

```diff
@@ -205,14 +205,19 @@ local function handle_buttons(this, fields)
 		local mods = worldfile:to_table()
 
 		local rawlist = this.data.list:get_raw_list()
+		local was_set = {}
 
 		for i = 1, #rawlist do
 			local mod = rawlist[i]
 			if not mod.is_modpack and
 					not mod.is_game_content then
 				if modname_valid(mod.name) then
-					worldfile:set("load_mod_" .. mod.name,
-						mod.enabled and "true" or "false")
+					if mod.enabled then
+						worldfile:set("load_mod_" .. mod.name, mod.virtual_path)
+						was_set[mod.name] = true
+					elseif not was_set[mod.name] then
+						worldfile:set("load_mod_" .. mod.name, "false")
```

Comment:
Can't we just not write these "false" lines?
They leak the names of all installed mods, which makes sending worlds to other persons a bit awkward.
And afaik their only real use was to provide defaults to edit if you want to enable mods without a gui.

Reply:
This is already done before this PR. This should be another issue and another PR